Makefile

```diff
@@ -1,18 +1,26 @@
-TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
+TOOL_BIN := bin/gotools/$(shell uname -s)-$(shell uname -m)
+PROTO_FILES := $(shell find proto/ -type f -name '*.proto')
 
-PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):`pwd`/node_modules/.bin:${PATH}"
+export GOBIN := $(shell pwd)/$(TOOL_BIN)
+export PATH := $(GOBIN):$(shell npm root)/.bin:$(PATH)
 
-PROTO_FILES=$(shell find proto/ -type f -name '*.proto')
+.PHONY: all
+all: clean build
```

Comment:
By convention, I think running `make` without arguments should run an `all` target, and a simple `clean build` seemed like the way to go with that